.circleci/config.yml

```diff
@@ -3,10 +3,104 @@ version: 2.1
 orbs:
   silta: silta/silta@0.1
 
+images:
+  - &node
+    image: circleci/node:12.16.1-browsers
+
+executors:
+  node:
+    docker:
+      - *node
+
+jobs:
+  install:
+    executor: node
+    steps:
+      - checkout:
+          path: ~/project/build_path
+      - restore_cache:
+          keys:
+            - v1-circleci-nextjs-dependecies-{{ .Branch }}-{{ .Revision }}
+      - run:
+          name: Install Dependencies
+          command: cd build_path && npm ci
+      - save_cache:
+          key: v1-circleci-nextjs-dependecies-{{ .Branch }}-{{ .Revision }}
+          paths:
+            - ~/.npm
+            - ~/.cache
+  jest:
+    docker:
+      - image: circleci/node:12.16.1-browsers
```

Comment:
Maybe we could use Node LTS version. v12 is quite old.
There is option to use v14.
https://circleci.com/developer/images/image/cimg/node#image-tags